docs/excel/custom-functions-runtime.md

```diff
@@ -1,13 +1,15 @@
 ---
-ms.date: 09/20/2018
+ms.date: 09/27/2018
 description: Excel custom functions use a new JavaScript runtime, which differs from the standard Add-ins WebView control runtime.
 title: Runtime for Excel custom functions
 ---
 
-# Runtime for Excel custom functions (Preview)
+# Runtime for Excel custom functions (preview)
 
 Custom functions extend Excel’s capabilities by using a new JavaScript runtime that uses a sandboxed JavaScript engine rather than a web browser. Because custom functions do not need to render UI elements, the new JavaScript runtime is optimized for performing calculations, enabling you to run thousands of custom functions simultaneously.
```

Comment:
So I spoke with Michael on these points yesterday - he thinks: 
a) it's not important that you can run thousands simultaneously (he feels this is marketing-y)
b) the sandboxed engine in only on desktop (otherwise, web still uses WebView runtime like normal)

Maybe we say something like "On Excel's desktop client, custom functions use a new, sandboxed JavaScript engine rather than a web browser."

Reply:
I will work on a PR today with some of his changes, but anyway, added this more as a heads up